Client/Frontend/Browser/Tabs/RemoteTabs/RemoteTabsPanel.swift

```diff
@@ -0,0 +1,379 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+import Storage
+import Common
+import Shared
+import Redux
+
+/// Captures state needed to populate the Sync tab UI. Some aspects of how we
+/// will handle management of state are still TBD as part of the Redux refactors.
+struct RemoteTabsPanelState {
```

Comment:
`RemoteTabsPanelState` struct should be in its own file following a similar folder struct as ThemeSettings